tests/unit/extensions/test_gunicorn.py

```diff
@@ -127,6 +127,83 @@ def test_flask_extension_default(tmp_path, flask_input_yaml, packages):
     }
 
 
+@pytest.mark.usefixtures("flask_extension")
+def test_flask_extension_async(tmp_path, flask_input_yaml):
```

Comment:
this test is almost identical to `test_flask_extension_default`. For the sake of our collective sanity, can't we just do a single test with the proper `parametrize` parameters?

Reply:
some like (I didn't test this)
```python
@pytest.mark.usefixtures("flask_extension")
@pytest.mark.parametrize("packages", ["other\nflask", "flask", "Flask", " flask == 99"])
@pytest.mark.parametrize(("async_package", "expected_worker"), [("gevent", "gevent"),("", "sync")])
def test_flask_extension_default(tmp_path, flask_input_yaml, packages, async_package, expected_worker):
    full_packages = "\n".join([packages, async_package])